components/Header/index.js

```diff
@@ -18,15 +19,15 @@ const handleClickCategories = () => {
 
 const Header = ({ backgroundColour, imgSrc, imgAlt, title, caption }) => {
```

Comment:
Really minor point on spelling - almost everything in the coding world seems to use the american spelling of "color", so it may be worth going with `backgroundColor` for the variable name. [It means you can use object shorthand where possible as well then ](https://alligator.io/js/object-property-shorthand-es6/)